.github/workflows/docker-iam-rucio-sync.yml

```diff
@@ -62,6 +45,14 @@ jobs:
         working-directory: containers/iam-rucio-sync
         run: |
           docker build . --tag  ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest
-          docker run ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest
           docker push  ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest
+          export sha=${{ github.sha }} 
+          export short_sha=${sha:0:7}
```

Comment:
Could you point me out the env var pointing to the short sha, please ? I couldn't find it. 
About the `version.txt` I don't see any advantage of having a file instead of fetching the version directly from the repository tag. But in general, yes I agree with the approach of the tagging, of course

Reply:
I was mistaken, there is no short SHA.
You're right let's work with the automated SHA and try the repo versioning you suggested (ref https://github.com/vre-hub/project/issues/33).

Reply:
for info, I will be using `git describe` that contains the version, the number of commits from a last tagged version, plus the short sha, so basically just useful info.